google-cloud-tools-plugin/resources/config.properties

```diff
@@ -19,3 +19,5 @@ feature.appengine.flex=true
 
 # Should be used in any Notifications explicitly produced by the plugin.
 notifications.plugin.groupdisplayid=Google App Engine plugin
+
+cloudsdk.min.version=131.0.0
```

Comment:
Is that part of the java style guide? (Though I'm not sure if that would apply to properties files).  This is something that I don't really feel strongly about, but I'm not sure this is really worth changing.